frontend/awx/interfaces/Inventory.ts

```diff
@@ -1,6 +1,72 @@
-import { Inventory as SwaggerInventory } from './generated-from-swagger/api';
+import { SummaryFieldsByUser, SummaryFieldsOrganization } from './summary-fields/summary-fields';
+import { Label } from './Label';
 
-export interface Inventory extends Omit<SwaggerInventory, 'id' | 'name' | 'summary_fields'> {
```

Comment:
This removes the usage of the interface that is generated from swagger.
Whats the thoughts behind this?